app/view/toolbar/CircleSelectionTool.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * This class is the circle selection of cpsi mapview application
+ * It can be used e.g. for the zone tool
+ */
+/**
+ * Circle Selection
+ *
+ * @class CpsiMapview.view.toolbar.CircleSelection
+ */
+Ext.define('CpsiMapview.view.toolbar.CircleSelection', {
```

Comment:
The class name should reflect the the JS file incl. its path:

Due to the path of the source file the class should be named the following: `CpsiMapview.view.toolbar.CircleSelectionTool`

Reply:
solved in https://github.com/compassinformatics/cpsi-mapview/pull/177/commits/b79cde00e7b4d84cc737d59866c7570cad768ee5